ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/fulu/helpers/MiscHelpersFulu.java

```diff
@@ -238,21 +239,34 @@ public List<Bytes32> computeDataColumnKzgCommitmentsInclusionProof(
         beaconBlockBody.getBackingNode(), getBlockBodyKzgCommitmentsGeneralizedIndex());
   }
 
-  public List<DataColumnSidecar> constructDataColumnSidecars(
+  @VisibleForTesting
+  @Deprecated
+  public List<DataColumnSidecar> constructDataColumnSidecarsOld(
```

Comment:
This is only used by tests now. Can we get rid of it?

Reply:
In separate pr please